crate/multiworld-gui/src/main.rs

```diff
@@ -1418,6 +1427,17 @@ impl Application for State {
             Message::ShowLoggingInstructions => if let Err(e) = open("https://github.com/midoshouse/ootr-multiworld/blob/main/assets/doc/logging.md") {
                 return cmd(future::err(e.into()))
             },
+            #[cfg(windows)]
+            Message::LaunchProject64 => {
+                    let emulator_path = self.pj64_script_path.as_ref().expect("emulator path must be set for Project64 version 3");
+                    let Some(pj64_folder_path) =  Path::new(emulator_path).ancestors().nth(2) else {
```

Comment:
Sneaky double space:

```suggestion
                    let Some(pj64_folder_path) = Path::new(emulator_path).ancestors().nth(2) else {
```

Reply:
good catch, i did try to format the file, but i there were too many changes so i did formating manually, i missed that one.
i removed it 👍 
do you have a lint/format config file? or you do all the formating manually?

Reply:
I do the formatting manually because I've had bad experiences with rustfmt doing things like hockey stick indentation, splitting a `use` item into multiple ones, adding semicolons after diverging expressions, or completely breaking macro invocations.